src/osmf/lib/HDSPlayer.as

```diff
@@ -430,6 +432,7 @@ package
       if (_playerState == MediaPlayerState.READY || _playerState == MediaPlayerState.PAUSED 
           || _playerState == MediaPlayerState.BUFFERING)
       {
+        _playCompleted = false;
```

Comment:
Hi Angie,

onVideoPlay event Initiates the play functionality through the plugin.

[image: Inline image 1]

Because of this reason we placed _playCompleted = false in onVideoPlay
event.

Thanks
Sarita Singh

On Mon, Apr 18, 2016 at 11:31 AM, angieisrani <notifications@github.com>
wrote:

> In src/osmf/lib/HDSPlayer.as
> <https://github.com/ooyala/html5-video-plugins/pull/144#discussion_r60110226>
> :
>
> > @@ -430,6 +432,7 @@ package
> >        if (_playerState == MediaPlayerState.READY || _playerState == MediaPlayerState.PAUSED
> >            || _playerState == MediaPlayerState.BUFFERING)
> >        {
> > +        _playCompleted = false;
>
> this looks good but i wonder also if this may be good to set to false when
> the video url changes but before play is called
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ooyala/html5-video-plugins/pull/144/files/643510f4579985f3bbc2bdf889d103e4333a60cb#r60110226>
>



-- 
Best Regards,
Sarita Singh